sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -112,16 +113,60 @@ protected SchemaTransform from(KafkaReadSchemaTransformConfiguration configurati
     consumerConfigs.put(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG, 100);
     consumerConfigs.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, autoOffsetReset);
 
+    String format = configuration.getFormat();
+
+    if (format == null || format.isEmpty()) {
+      format = "raw";
```

Comment:
Do we want to add some sort of log for the user, so that they know that RAW wound up auto-set?

Reply:
Do we think that RAW is the ideal default?  

I am thinking ahead to a future where we might be able to pickup information from the schema shared to then choose to set the format.  That's probably outside the scope of this PR, but to think of a possible future where that is possible.

Reply:
I think we should require the user to set this. In the future we may want to pull the appropriate schema from metadata (in that case we could add an explicit "infer" value if we wanted, but if there is a default that should be it). We should leave ourselves options--we can always set a default later if we really want but we can't retract one.